docs_source_files/content/introduction/types_of_testing.ja.md

```diff
@@ -1,120 +1,78 @@
 ---
-title: "Types of testing"
+title: "テストの種類"
 weight: 3
 ---
 
-{{% notice info %}}
-<i class="fas fa-language"></i> ページは英語から日本語へ訳されています。
-日本語は話せますか？プルリクエストをして翻訳を手伝ってください!
-{{% /notice %}}
 
-## Acceptance testing
-This type of test is done to determine if a product's
-feature matches its requirements.
-This generally involves the customer's feedback or specification.
+## 受入テスト
+このタイプのテストは、製品の機能が要件に一致するかどうかを判断するために行われます。
+これには通常、顧客のフィードバックまたは仕様が含まれます。
 
-For web applications, the automation of this testing can be done
-directly with Selenium by simulating user expected behavior.
-This simulation could be done by record/playback or through the
-different supported languages as explained in this documentation.
-Note: Acceptance testing is a subtype of **_functional testing_**,
-which some people might also refer to.
+Webアプリケーションの場合、ユーザーの予想される動作をシミュレートすることで、このテストの自動化をSeleniumで直接実行できます。
+このシミュレーションは、このドキュメントで説明されているように、記録/再生によって、またはサポートされているさまざまな言語によって実行できます。
+注：受入テストは **機能テスト** のサブタイプであり、一部の人はこれも参照する場合があります。
 
-### Functional testing
-This type of test is done to determine if a product's
-feature functions properly, without issues.
+### 機能テスト
+このタイプのテストは、製品の機能が問題なく正常に機能するかどうかを判断するために行われます。
 
-This generally include: the tests work without errors
-(404, exceptions...), in an usable way (right redirections),   
-in an accessible way and matching its specifications            
-(see **_acceptance testing_** above).
+これには通常、エラーなし（404、例外…）、使用可能な方法（正しいリダイレクト）、アクセス可能な方法、およびその仕様に一致するテスト（上記の **受入テスト** を参照）が含まれます。
 
-For web applications, the automation of this testing can be
-done directly with Selenium by simulating expected returns.     
-This simulation could be done by record/playback or through     
-the different supported languages as explainedin this documentation.
+Webアプリケーションの場合、期待されるリターンをシミュレートすることにより、このテストの自動化をSeleniumで直接実行できます。
+このシミュレーションは、このドキュメントで説明されているように、記録/再生またはサポートされているさまざまな言語で実行できます。
 
-### Performance testing
-As its name indicates, performance tests are done 
-to measure how well an application is performing.
+### パフォーマンステスト
+その名前が示すように、パフォーマンステストは、アプリケーションのパフォーマンスを測定するために行われます。
 
-There are two main sub-types for performance testing:
+パフォーマンステストには2つの主なサブタイプがあります。
 
-#### Load testing
-Load testing is done to verify how well the
-application works under different defined loads
-(usually a particular number of users connected at once)
+#### ロードテスト
```

Comment:
Hi @kzhirata ,

When I translate with third-party tools, the word "ロードテスト" gives me road test instead of the 'load test' can you please cross-check.

Apologies if am wrong, just want to make sure of translated docs.

Reply:
Thenk you for reviewing, @Harsha509 .
In Japanese, the both "road test" and "load test" use the same letters "ロードテスト".

http://e-words.jp/w/%E6%80%A7%E8%83%BD%E3%83%86%E3%82%B9%E3%83%88.html
http://yougo.ascii.jp/caltar/%E3%83%AD%E3%83%BC%E3%83%89%E3%83%86%E3%82%B9%E3%83%88